build.cmd

```diff
@@ -1,7 +1,7 @@
 @echo off
 setlocal
 
-set _args=-restore -build /p:_SelfContainedWasSpecified=true %*
```

Comment:
The original code here was `/p:TreatWarningsAsErros=true`. This is intentionally being removed because it's redundant with the defaults for `$warnAsError` of `build.ps1` and together it makes it hard to turn this off locally.